### ISSUE TITLE
question - account setting - username 

### ISSUE THREAD

**Issue by Lesley2** [Acfee/ehna-dev#23]
So I uploaded my general information - then clicked on account and their is an option to change my username which the website says is @bGVzbGV5LmRvYnNvbkB0. if people don't click on account setting - which I'm guessing most wont - will that be their username? 
![snip20170314_124](https://cloud.githubusercontent.com/assets/26297259/23892698/f5bcd8ec-089b-11e7-959d-969fe8952a79.png)

**Comment by getwes** [comment]
@Lesley2 @SBroomhead 

Just to give an overview of my original idea for v4 registration was that a user could "choose a username" apon his account setup process. 

Later in the dev myself and @SBroomhead both agreed that registration should only carry a "email address" and "create password" which then simplified the registration process and a user could immediately begin using ehna. 

So regarding that outcome the system would automatically generate you a username. however the username is important as its a unique identifier to who you are. 

Example
> https://ehna.acfee.org/lesley.dobson

So this is a big deal regarding how the system recognizes you and deals with information regarding who you are. 

In short. Yes that is their username until they start personalizing their settings.

**Comment by SBroomhead** [comment]
@getwes I am still convinced we are on the right track with the very slick and easy registration. However, for the username we create, we spoke about creating the auto-created username as a combination of the names provided by the user. It might not be perfect, but it would be closer than a random code. Is that still possible?

**Comment by getwes** [comment]
Used email address as username for initial sign up process
